src/components/Text/SearchInstructions.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+//import classes from '../Search/Search.module.css';
```

Comment:
Never use different component's styling module (Search.module.css) for styling of this component (SearchInstructions.js)! Always create styling component which is scoped only to this component! That is the reason we use module.css concept, because css classes from .module.css are not global, but scoped only to the component for which they are dedicated.

Reply:
Done.

Reply:
ok